plugins/header_rewrite/operators.h

```diff
@@ -109,16 +109,17 @@ class OperatorSetRedirect : public Operator
 public:
   OperatorSetRedirect() { TSDebug(PLUGIN_NAME_DBG, "Calling CTOR for OperatorSetRedirect"); }
   void initialize(Parser &p) override;
+
   TSHttpStatus
-  get_status()
+  get_status() const
   {
     return static_cast<TSHttpStatus>(_status.get_int_value());
```

Comment:
Heh. You could see if `return { _status.get_int_value() };` works.

Reply:
Didn't compile :-/

``
header_rewrite/operators.h:116:13: error: cannot initialize return object of type 'TSHttpStatus' with an rvalue of type 'int'
    return {_status.get_int_value()};
            ^~~~~~~~~~~~~~~~~~~~~~~
``